opus/application/apps/cart/views.py

```diff
@@ -812,27 +832,28 @@ def _get_download_info(product_types, session_id):
     sql += q('obs_files')+'.'+q('sort_order')+' AS '+q('sort')+', '
     sql += q('obs_files')+'.'+q('short_name')+' AS '+q('short')+', '
     sql += q('obs_files')+'.'+q('full_name')+' AS '+q('full')+', '
-    sql += q('obs_files')+'.'+q('default_checked')+' AS '+q('checked')
+    sql += q('obs_files')+'.'+q('default_checked')+' AS '+q('checked')+', '
+    sql += q('obs_files')+'.'+q('version_name')+' AS '+q('ver')
     sql += 'FROM '+q('obs_files')+' '
     sql += 'INNER JOIN '+q('cart')+' ON '
     sql += q('cart')+'.'+q('obs_general_id')+'='
     sql += q('obs_files')+'.'+q('obs_general_id')+' '
     sql += 'WHERE '+q('cart')+'.'+q('session_id')+'=%s '
     values.append(session_id)
-    sql += 'AND '+q('obs_files')+'.'+q('version_number')+' >= 900000 '
-    sql += 'ORDER BY '+q('sort')
+    # Put "Current" version on top of others
+    sql += 'ORDER BY '+q('sort')+', FIELD('+q('ver')+' ,"Current") DESC'
```

Comment:
Can you just say `ver == 'Current'` (and let it sort on True/False) instead of using the `FIELD` function, which is probably slower?